build.gradle

```diff
@@ -34,6 +34,7 @@ ext {
     netLingalaZip4jVersion = "2.3.2"
     commonsIoVersion = "2.6"
     commonsLang3Version = "3.9"
+    socketVersion="1.3.0-SNAPSHOT"
```

Comment:
Is this version correct? If we are using the `ballerina-lang` version, this should be `2.0.0-SNAPSHOT`, otherwise we should use the socket module version.

Reply:
Set the socket version to the project version. Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/482/commits/fea47d5b74fcc893a1d5d53267638e2fc193a277